### ISSUE TITLE
Do not use inspect.getargspec() in twisted/python/usage.py on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8705]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8705|
|Type|enhancement|
|Created|2016-07-28 21:39:50Z|
|Branch|https://github.com/twisted/twisted/tree/8705-rodrigc-getargspec-py3|

inspect.getargspec() is deprecated on Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8705 8705
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8705_rodrigc_getargspec_py3 8705-rodrigc-getargspec-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469741990243520 1469741990243520
changetime__1470345713821356 1470345713821356
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/442

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa set owner to @rodrigc|
|-|-|

LGTM, merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 3e488bca70159bd8d7f5a62dc55e0ab9879f229b
```
#!CommitTicketReference repository="" revision="3e488bca70159bd8d7f5a62dc55e0ab9879f229b"
Merge pull request #442 from twisted/8705-rodrigc-getargspec-py3

Author: rodrigc
Reviewer: Lukasa
Fixes: #8705

Do not use inspect.getargspec() in twisted/python/usage.py on Python 3
```
